createCustomModel.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import json
+from config import Config
+from ibm_watson import TextToSpeechV1
+from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
+
+# python createCustomModel.py config.ini "script test" en-US "test the script for creating a new model"
+
+config_file = "config.ini"
```

Comment:
For a few of these model scripts, we can consolidate to one like we did in STT-WER-Python-Tool: https://github.com/IBM/watson-stt-wer-python/blob/main/models.py

Reply:
File is removed.